utils/connectivity/conn_http.c

```diff
@@ -27,9 +27,31 @@ void mbedtls_debug(void *ctx, int level, const char *file, int line, const char
   fflush((FILE *)ctx);
 }
 
+static inline status_t free_info_context(connect_info_t *const info) {
```

Comment:
Don't declare as `inline`. Use plain `static` instead. Compiler optimizations can do in efficient ways.